examples/listkeys.c

```diff
@@ -83,6 +83,7 @@ int main(int argc, char *argv[])
 	list_keys("Public keys", keys, nkeys);
 
 	if (slot->token->loginRequired && argc > 2) {
+		password = malloc(sizeof(argv[2])+1);
```

Comment:
s/sizeof/strlen 
... else the fix will be worst than the initial version...
Where/when will it be free ?

Reply:
You are correct. Sorry for the mistake.

I guess one could free after memset in line 92?